hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/codec/FileServiceStateTranslator.java

```diff
@@ -63,11 +63,17 @@ public static File stateToPbj(
         requireNonNull(metadata);
         requireNonNull(fileID);
         final var fileBuilder = new File.Builder();
-        fileBuilder.fileNumber(fileID.getFileNum());
+        fileBuilder.fileId(new FileID.Builder()
+                .fileNum(fileID.getFileNum())
+                .realmNum(fileID.getRealmNum())
+                .shardNum(fileID.getShardNum()));
         fileBuilder.expirationTime(metadata.getExpiry());
-        if (metadata.getWacl() != null)
+        if (metadata.getWacl() != null) {
             fileBuilder.keys(PbjConverter.asPbjKey(metadata.getWacl()).keyList());
-        if (data != null) fileBuilder.contents(Bytes.wrap(data));
+        }
+        if (data != null) {
```

Comment:
Our style guide allows us to leave out the braces here (and the line above).
>### 4.1.1 Use of optional braces
>Single-line if statements are permitted even without braces.

Reply:
Tell that to Sonar. I added the braces because SonarLint was complaining.

Reply:
Sonar lint is wrong (I basically ignore sonar at this point, it's way off from what spotless wants, and sometimes making sonar not complain causes spotless to fail).
It's a minor item, and the braces are permitted, but it's also reasonable to leave them alone too.